src/sentry/static/sentry/app/views/settings/organization/projects/organizationProjectsView.jsx

```diff
@@ -29,9 +31,29 @@ class OrganizationProjectsView extends OrganizationSettingsView {
 
   renderBody() {
     let {projects} = this.props;
+    let {organization} = this.context;
+    let {access} = organization;
+    let canCreateProjects = access.indexOf('org:write') > -1;
```

Comment:
hmm i think admins can create new teams and projects but they don't have org write: https://github.com/getsentry/sentry/blob/master/src/sentry/conf/server.py#L1101

Reply:
maybe `project:admin` ?